website/pages/blog/product-updates-1.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "CloudQuery Product Updates #1"
+tag: product
+date: 2022/11/01
+description: >-
+  Monthly updates on CloudQuery product and roadmap.
+author: yevgenypats
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+Hey everyone! Since CloudQuery [V1 is out](./cloudquery-v1-release.md) we will be restarting our monthly products updates and starting from #1 as figured it easier to use number then months :)
+
+This months we've been really cranking it at CQ so lot's of good stuff!
+
+## CLI Imprvoments
+
+* We've introduced glob to specify tables. Check out release [blog](https://www.cloudquery.io/blog/introducing-wildcard-matching-for-tables)
+
+## Destinations
+
+We've added two new destinations to CQ!
+
+* [CSV](https://github.com/cloudquery/cloudquery/blob/main/plugins/destination/csv/README.md). See Release [Blog](https://www.cloudquery.io/blog/scaling-out-elt-with-cq-and-csv)
+* [SQLite](https://github.com/cloudquery/cloudquery/tree/main/plugins/destination/sqlite). See Release [Blog](https://www.cloudquery.io/blog/announcing-cloudquery-sqlite-destination)
+
+
+## Sources
+
+### AWS
+
+We've added 16 New Service!
+
+* [EventBridge Schedule](https://github.com/cloudquery/cloudquery/issues/4693)
+* [SSM Resources](https://github.com/cloudquery/cloudquery/issues/4381)
+* [ECS Scale](https://github.com/cloudquery/cloudquery/issues/4688)
+* [MSK](https://github.com/cloudquery/cloudquery/issues/3967)
+* [AWS RAM](https://github.com/cloudquery/cloudquery/issues/3961) 
+* [AWS Image Scan Findings](https://github.com/cloudquery/cloudquery/issues/3607)
+* [ECR Repository Policy](https://github.com/cloudquery/cloudquery/issues/3844)
+* [Appstream](https://github.com/cloudquery/cloudquery/issues/3567)
+* [AppRunner Additions](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/aws/CHANGELOG.md#460-2022-11-06)
+* [KMS Key Grants](https://github.com/cloudquery/cloudquery/issues/3441)
+* [MWAA](https://github.com/cloudquery/cloudquery/issues/3431)
+* [Service Quotas](https://github.com/cloudquery/cloudquery/issues/3489)
+* [Fraud Detector Support](https://github.com/cloudquery/cloudquery/issues/3076)
+* [Service Catalog](https://github.com/cloudquery/cloudquery/issues/3275)
+* [App Runner](https://github.com/cloudquery/cloudquery/issues/2997)
+* [Neptune](https://github.com/cloudquery/cloudquery/issues/2923)
```

Comment:
I haven't checked, but from memory I think `Eventbridge` is missing from this list? (It's separate from `Eventbridge Scheduler`)